server/lib/configuration.js

```diff
@@ -149,6 +149,33 @@ var conf = module.exports = convict({
     doc: 'Google Analytics id',
     format: String
   },
+  hpkp: {
+    enabled: {
+      default: true,
```

Comment:
Similar to the below comment about removing the SHAs, let's set this to false by default, and override for prod/stage/fxa-dev. Enabling this feature with incorrect values can cause big problems, let's guard our future selves against our current selves.